clients/src/test/java/org/apache/kafka/clients/admin/AdminClientUnitTestEnv.java

```diff
@@ -82,7 +98,18 @@ public void update(Time time, MockClient.MetadataUpdate update) {
         });
 
         metadataManager.update(cluster, time.milliseconds());
+        if (createAdminClient)
+            this.adminClient = KafkaAdminClient.createInternal(adminClientConfig, metadataManager, mockClient, time);
+    }
+
+    /**
+     * Initiates the AdminClient
+     */
+    public AdminClientUnitTestEnv initiate() {
```

Comment:
nit: Perhaps call this method `createAdminClient()` to be consistent with the flag in the constructor?